src/comp_loinc/cli.py

```diff
@@ -26,29 +26,29 @@ def __init__(self):
         self.cli.callback(invoke_without_command=True)(self.callback)
         self.cli.command()(self.parts_list)
 
-    def callback(self, /, *,
-                 work_dir: t.Annotated[t.Optional[Path], typer.Option(help='CompLOINC work directory.')] = Path.cwd(),
-                 loinc_release: t.Annotated[t.Optional[Path], typer.Option(
-                     help=f'Path to a directory containing an unpacked LOINC release. Defaults to: ./{LOINC_RELEASE_DIR_NAME}')] = None,
-                 loinc_trees: t.Annotated[t.Optional[Path], typer.Option(
-                     help=f'A directory name to load tree files from. Defaults to {LOINC_TREES_DIR_NAME}')] = None,
-
-                 loinc_version: t.Annotated[t.Optional[str], typer.Option(
-                     help='The LOINC release version. It uses the directory name of the the loinc_release option by if it appears to be a release number.')] = None,
-
-                 out_dir: t.Annotated[t.Optional[Path], typer.Option(help='The CompLOINC output directory.')] = None,
-                 log_level: t.Annotated[str, typer.Option(help='Logging level. Defaults to WARN.')] = 'WARN',
-                 owl_output: t.Annotated[bool, typer.Option()] = True,
-                 rdf_output: t.Annotated[bool, typer.Option()] = True
-                 ):
-
+    def callback(
+        self, /, *,
```

Comment:
## CLI Questions
1. What is the purpose of this, along with`main.py`, which also has a CLI?
  - I think that `main.py` is the original implementation, and I'm assuming that this is the new one, and is a work in progress, and that we'll delete `main.py` afterwards?
2. Is the class-based approach a standard Typer pattern?
  - It looks a bit Java-esque. I personally find it slightly verbose and confusing (at first glance).